recipes/simwalk/meta.yaml

```diff
@@ -0,0 +1,38 @@
+package:
+  name: simwalk2
+  version: "2.91"
+
+source:
+  fn: SimWalk2-2.91-lin.tar.gz # [linux]
+  url: https://github.com/BioTools-Tek/packages/blob/master/simwalk2/SimWalk2-2.91-lin.tar.gz?raw=true # [linux]
+  md5: 2a529d50ca48cd7ea2b67f85c290f203 # [linux]
+
+  fn: SimWalk2-2.91-osx.tar.gz # [osx]
+  url: https://github.com/BioTools-Tek/packages/blob/master/simwalk2/SimWalk2-2.91-osx.tar.gz?raw=true # [osx]
+  md5: 157cf4a869afdacf8e415005886f27c7 # [osx]
+
+  
+build:
+  number: 0
+  #skip: True # [osx]
+
+requirements:
+  build:
+    - libgfortran # [linux]
```

Comment:
This means you only have dependencies for Linux and no dependencies at all for OSX.
You also want to have GCC as build dependency and not libgcc, libgcc is runtime dependency.

Reply:
will fix, thanks